tests/src/coffee_httpd_integration_tests.rs

```diff
@@ -67,3 +67,241 @@ pub async fn httpd_init_add_remote() {
 
     cln.stop().await.unwrap();
 }
+
+#[tokio::test(flavor = "multi_thread")]
+#[ntest::timeout(560000)]
+pub async fn httpd_add_remove_plugins() {
+    init();
+
+    let mut cln = Node::tmp("regtest").await.unwrap();
+    let lightning_dir = cln.rpc().getinfo().unwrap().ligthning_dir;
+    let lightning_dir = lightning_dir.strip_suffix("/regtest").unwrap();
+    let manager = CoffeeHTTPDTesting::tmp(lightning_dir.to_string())
+        .await
+        .unwrap();
+    log::info!("lightning path: {lightning_dir}");
+    let url = manager.url();
+    log::info!("base url: {url}");
+    let client = reqwest::Client::new();
+
+    // Define the request body to be sent to the /remote/add endpoint
+    let remote_add_request = RemoteAdd {
+        repository_name: "lightningd".to_string(),
+        repository_url: "https://github.com/lightningd/plugins.git".to_string(),
+    };
+
+    // Send the request to add a remote repository
```

Comment:
The comment is obvious so you can avoid to add the commit everywhere

The comment in the test is added when you are doing a not obvious things 

Change all occurrence in the tests